lewis/core/devices.py

```diff
@@ -112,16 +112,9 @@ def __init__(self, module):
 
         self._device_types = list(get_members(self._module, is_device).values())
 
-        if not self._device_types:
-            # syslog.warning('No devices discovered in module {}.'.format(module.__name__))
-            pass
-        else:
-            pass
-            # syslog.info('Discovered devices in module {}: {}'.format(
-            #    module.__name__, ','.join([x.__name__ for x in self._device_types])))
-
         submodules = get_submodules(self._module)
 
+        self._device_types += list(get_members(submodules.get('device'), is_device).values())
```

Comment:
Do you think `states.py` and (for the chopper) `bearings.py` should also go into the `device/` directory?

That would be my gut instinct. Then we have a separation between behaviour (everything in `device/`) and interface (everything in `interfaces/`). Also it mirrors the control interface nicely.

`device/` or `devices/` (to match `interfaces/`, `setups/`, `tests/`, ...)?